test/types/range/ferguson/range-split-inits.chpl

```diff
@@ -0,0 +1,32 @@
+proc test1() {
+  writeln("test1");
+  var a:range;
+  writeln(a.type:string, " ", a);
+
+  var b;
+  b = 1..10;
+  writeln(b.type:string, " ", b);
+  
+  var c:range(?);
+  c = 2..20 by 2;
+  writeln(c.type:string, " ", c);
+  
+  var d:range;
+  d = 3..30;
+  writeln(d.type:string, " ", d);
+}
+test1();
+
+// If these become errors, split them into a separate test.
+// Here a variable declared with type `range` is initialized
+// to a non-default range.
+proc test2() {
```

Comment:
Aren't these supposed to be errors?

Reply:
Hmm, I thought these were errors on master, but I guess not.

Reply:
Right - this is the topic of issue and my recent comments there #13566. I'm viewing fixing that bug as a separate effort (since it happens for variable initialization, not just out intents).